src/engine/v3/V3Interpreter.v3

```diff
@@ -910,6 +910,7 @@ component V3Interpreter {
 				var high = codeptr.read_u64();
 				push(Value.V128(low, high));
 			}
+			V128_XOR => do_vv_v(V3Eval.V128_XOR);
```

Comment:
Please go ahead and do all the `i64x2` pure operations to save on the number of PRs.

Reply:
Got it. Just want to make sure I'm on the right track. Further PRs will have multiple realizations as before.